_includes/footer.html

```diff
@@ -1,66 +1,169 @@
 <footer class="usa-footer site-footer" role="contentinfo">
-  <div class="usa-identifier site-identifier">
-    <section class="usa-identifier__section usa-identifier__section--masthead" aria-label="Agency identifier">
-        <div class="usa-identifier__container">
-            <div class="usa-identifier__logos">
-                <a href="https://www.gsa.gov">
-                    {% image_with_class "_img/gsa-logo.svg" "usa-identifier__logo" "GSA logo identifier" %}
-                </a>
+    <div class="footer-section-bottom usa-footer__primary-section bg-accent-warm-light border-top border-accent-warm-dark">
+        <div class="grid-container  padding-bottom-4">
+            <div class="grid-row grid-gap-4">
+                <div class="mobile-lg:grid-col-6 desktop:grid-col-3">
+                    <section class="usa-footer__primary-content usa-footer__primary-content--collapsible">
+                        <h4 class="usa-footer__primary-link">Get in touch</h4>
+                        <ul class="usa-list usa-list--unstyled">
+                            <li class="usa-footer__secondary-link"><a href="/contact">Customer support</a></li>
```

Comment:
would you be willing to put these footer links into a yaml/json/whatever list in the data folder and loop through them rather than hardcode the link text and hrefs into the template? I think it'd be an improvement from a content management perspective.